lib/matplotlib/font_manager.py

```diff
@@ -899,7 +899,9 @@ def set_size(self, size):
             size = float(size)
         except ValueError:
             if size is not None and size not in font_scalings:
-                raise ValueError("size is invalid")
+                raise ValueError(
+                    "Size is invalid. Valid font size are" + ", ".join(
```

Comment:
super picky: I think you need a trailing space after 'are' and I don't think you need the `[]` in the generator.